test/lit/passes/remove-unused-module-elements-refs.wast

```diff
@@ -96,11 +64,8 @@
   )
 
   ;; CHECK:      (func $target-B (type $B)
-  ;; CHECK-NEXT:  (unreachable)
+  ;; CHECK-NEXT:  (nop)
```

Comment:
Why did these change?

Reply:
Now that we're no longer passing --closed-world, we assume the reference to `target-B` may be called from the outside, so we no longer empty the function. I'll move this test to a separate file where we can still pass --closed-world.